community/CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# Contributing to SourceCred
+
+Contributing to SourceCred means any of the following:
+
+ - opening an issue to ask a question, or request a feature
+ - open a pull request (PR) to request for review
+
+See the [glossary](glossary.md) or [introduction to github guide](../guides/GITHUB.md) 
+if you have questions about terms used in this document.
+
+## Scope of Contribution
+
+This contribution guide covers the repositories (projects) under the 
+[SourceCred GitHub organization](https://github.com/sourcecred). While this contributing guide
+is generally applicable, each repository can potentially be different, and it's
+recommended to check within the repository you are interested
+in contributing to for a more detailed guide to contributing.
+
+## Opening Issues
+
+You can open an issue on any SourceCred project to ask a question, suggest a feature,
+or report a bug. If you want to have discussion before opening the issue, you are welcome 
+and encouraged to chat with others on [discourse](discourse.sourcecred.io/h), [discord](https://discordapp.com/channels/453243919774253079/670490733077332000), or any other method that is available and comfortable for you
+and the community. If you decide that an issue is warranted, the following steps
+are recommended.
+
+### 1. Clarify your question or need
+
+In the case of having a question, do your best to articulate the question, your particular
```

Comment:
Hmm, should we be guiding people to ask questions on specific GitHub repositories? I'd rather have a support category on Discourse.

Reply:
I'll add the #git-help mention here as well.

Reply:
The section here exists because it is specifically describing contributing via issues, and there are several cases that a user might think it most appropriate to ask a question in an issue and not GitHub (e.g., linking to other GitHub artifacts, mentions of lines, or being able to assign someone to something). I'll open the entire section, however, with a mention of discord so a user that is ambivalent will likely stop reading and click the link there.